### ISSUE TITLE
Add-on does not start after installing HA 2023.11

### ISSUE THREAD

**Issue by dutch-erik** [brenner-tobias/addon-cloudflared#518]
### The problem

Add-on does not start

### What version of Cloudflared has the issue?

4.2.9

### What was the last working version of Cloudflared?

4.2.9

### What type of installation are you running?

Home Assistant Supervised

### Anything in the logs that might be useful for us?

```txt
-----------------------------------------------------------
 Add-on: Cloudflared
 Use a Cloudflare Tunnel to remotely connect to Home Assistant without opening any ports
-----------------------------------------------------------
 Add-on version: 4.2.9
 You are running the latest version of this add-on.
 System: Home Assistant OS 11.1  (aarch64 / raspberrypi3-64)
 Home Assistant Core: 2023.11.0
 Home Assistant Supervisor: 2023.10.1
-----------------------------------------------------------
 Please, share the above information when looking for help
 or support in, e.g., GitHub, forums or the Discord chat.
-----------------------------------------------------------
Log level is set to DEBUG
[17:20:11] DEBUG: Cloudflared log level set to "info"
[17:20:11] DEBUG: Checking connectivity to Cloudflare
[17:20:11] DEBUG: Checking region1.v2.argotunnel.com TCP port 7844
[17:20:11] DEBUG: Checking region1.v2.argotunnel.com UDP port 7844
[17:20:12] DEBUG: Checking region2.v2.argotunnel.com TCP port 7844
[17:20:13] DEBUG: Checking region2.v2.argotunnel.com UDP port 7844
[17:20:14] DEBUG: Checking api.cloudflare.com TCP port 443
[17:20:14] INFO: Checking add-on config...
[17:20:17] DEBUG: Checking host {"hostname":"xxx.xxx.net","service":"https://192.168.2.210:5001"}...
[17:20:18] DEBUG: Checking host {"hostname":"yyy.xxx.net","service":"https://192.168.2.210:5006"}...
[17:20:22] INFO: Checking for existing certificate...
[17:20:22] INFO: Existing certificate found
[17:20:22] INFO: Checking for existing tunnel...
[17:20:22] INFO: Existing tunnel with ID 702a5225-c87f-409b-8293-bec52bc15aa8 found
[17:20:22] INFO: Checking if existing tunnel matches name given in config
2023-11-02T16:20:39Z WRN Your version 2023.8.2 is outdated. We recommend upgrading it to 2023.10.0
REST request failed: Get "https://api.cloudflare.com/client/v4/accounts/8fe9331f2117b484c22a8c26cc/cfd_tunnel?is_deleted=false&uuid=702a5225--8293-bec52bc15aa8": net/http: request canceled (Client.Timeout exceeded while awaiting headers)
[17:20:39] DEBUG: Existing Cloudflare Tunnel name: 
[17:20:39] ERROR: Existing Cloudflare Tunnel name does not match add-on config.
[17:20:39] ERROR: ---------------------------------------
[17:20:39] ERROR: Add-on Configuration tunnel name: ha_tunnel_1
[17:20:39] ERROR: Tunnel credentials file tunnel name: 
[17:20:39] ERROR: ---------------------------------------
[17:20:39] ERROR: Align add-on configuration to match existing tunnel credential file
[17:20:39] ERROR: or re-install the add-on.
s6-rc: warning: unable to start service init-cloudflared-config: command exited 1
/run/s6/basedir/scripts/rc.init: warning: s6-rc failed to properly bring all the services up! Check your logs (in /run/uncaught-logs/current if you have in-container logging) for more information.
/run/s6/basedir/scripts/rc.init: fatal: stopping the container.
```

### Steps to reproduce the issue

After installing new Home Assistant 2023.11.0 addon does not start anymore.

**Comment by roromix** [comment]
I have also the same problem today since the last HA update

**Comment by geirmala** [comment]
Same issue here, locked out from HA,

description: Use a Cloudflare Tunnel to remotely connect to Home Assistant without
    opening any ports
  detached: false
  homeassistant: null
  icon: true
  logo: true
  name: Cloudflared
  repository: 9074a9fa
  slug: 9074a9fa_cloudflared
  stage: stable
` **state: error**
  update_available: false
  url: https://github.com/brenner-tobias/addon-cloudflared/
  version: 4.2.9
  version_latest: 4.2.9
- advanced: false
  available: true
  build: false

**Comment by brenner-tobias** [comment]
There is currently an issue with Cloudflare, see #517.

**Comment by kwkid** [comment]
Same issue here but i did not install 2023.11.
I did come across this note from Cloudflare stating they had a system-wide power outage...I figured this had something to do ) the addon not working...Currently waiting to see

[Cloudflare Notice](https://www.cloudflarestatus.com/)

Error Snippet

s6-rc: warning: unable to start service init-cloudflared-config: command exited 1
/run/s6/basedir/scripts/rc.init: warning: s6-rc failed to properly bring all the services up! Check your logs (in /run/uncaught-logs/current if you have in-container logging) for more information.
/run/s6/basedir/scripts/rc.init: fatal: stopping the container.
